pkg/kubelet/cm/cpumanager/cpu_assignment.go

```diff
@@ -293,9 +357,9 @@ func (a *cpuAccumulator) freeCores() []int {
 	return free
 }
 
-// Returns free CPU IDs as a slice sorted by sortAvailableCPUs().
+// Returns free CPU IDs as a slice sorted by sortAvailableCPUsPacked().
```

Comment:
nit: `sortAvailableCPUsPackedOrSpread`

Reply:
nice catch. I forgot to update the name in comment